tests/conftest.py

```diff
@@ -348,12 +379,24 @@ def net_cleanup(self):
             fds = re_space.split(l)
             if len(fds) > 1:
                 pname = fds[1].rstrip(":")
-                m = re.compile("(eth|lo|Bridge|Ethernet)").match(pname)
+                m = re.compile("(eth|lo|Bridge|Ethernet|vlan|inband)").match(pname)
                 if not m:
                     self.ctn.exec_run("ip link del {}".format(pname))
                     print("remove extra link {}".format(pname))
         return
 
+    def vct_connect(self, vctdata):
+        if "inband_address" in vctdata:
+            ifpair = vctdata["inband_intf_pair"]
+            ifname = vctdata["inband_intf"]
+            iaddr = vctdata["inband_address"]
+            self.vct.connect(ifname, ifpair, str(self.ctn_sw_pid))
```

Comment:
Can we feed in the bridge instead of the parent object self.vct to this object?

Reply:
dvs is passed everywhere in the test but vct is not passed. 
If at all there is a need to call into vct then only way is to make dvs.vct.foo().
this is what happens in dpb_setup_fixture() which calls dvs.restart or dvs.vct.restart based on if there is a vct  or not.

vctdata can be looked up by dvs through a call to vct.get_inband(ctnname)